openbas-api/src/main/java/io/openbas/rest/inject/InjectApi.java

```diff
@@ -505,6 +518,19 @@ public Inject scenarioInject(
     return injectRepository.findById(injectId).orElseThrow(ElementNotFoundException::new);
   }
 
+  @Transactional(rollbackFor = Exception.class)
+  @PutMapping(INJECT_URI + "/{scenarioId}/injects/{injectId}/bulk")
+  @PreAuthorize("isScenarioPlanner(#scenarioId)")
+  public Inject bulkUpdateInjectForScenario(
+      @PathVariable String scenarioId,
+      @PathVariable String injectId,
+      @Valid @RequestBody InjectInput input) {
+    Scenario scenario = this.scenarioService.scenario(scenarioId);
+    Inject inject = bulkUpdateInject(injectId, input);
+    this.scenarioService.updateScenario(scenario);
```

Comment:
Is it really needed ? Just from a shallow view, it looks like we get the scenario and update it right after but is it really having an effect ?

Reply:
nop, you are right :), ty!